src/iplant_groups/clients/grouper.clj

```diff
@@ -484,32 +484,42 @@
 
 ;; This is only available as a Lite request; ActAsSubject works differently.
 (defn- format-group-folder-privileges-lookup-request
-  [entity-type username group-or-folder-name]
-  (if-let [name-key (get {:group :groupName
+  [entity-type username group-or-folder-name params]
+  (if-let [name-key (get {:group  :groupName
                           :folder :stemName}
                          entity-type)]
     {:WsRestGetGrouperPrivilegesLiteRequest
-     {:actAsSubjectId username
-      :includeSubjectDetail "T"
-      :includeGroupDetail   "T"
-      name-key group-or-folder-name}}
+     (remove-vals nil? {:actAsSubjectId       username
+                        :includeSubjectDetail "T"
+                        :includeGroupDetail   "T"
+                        name-key              group-or-folder-name
+                        :subjectId            (:subject-id params)
+                        :subjectSourceId      (:subject-source-id params)
+                        :privilegeName        (:privilege params)})}
```

Comment:
This adds the additional filters to the request if they're present in the query string.